### PR TITLE
Fix Attempting to JIT compile method while in AOT only mode for Swizzled Apple Native Timings

### PR DESCRIPTION
**Pull request by ProRedCat**
Adds compiler hint of `UnmanagedFunctionPointer` to delegate methods 

Tests requried:
- Real iOS device as this does not appear on simulators

### CODE REVIEW

**Comment by ProRedCat** [comment]
Still procuring a real iOS device to test these changes. This should go out in the next release

**Comment by ProRedCat** [comment]
Tested on a real device and this appears to be fixed. Will put out a release shortly for this after review

File: Raygun4Maui/MauiRUM/EventTrackers/Apple/RaygunUiViewControllerObserver.cs [hunk 1]
```diff
@@ -1,14 +1,12 @@
-﻿
-using System.Runtime.CompilerServices;
-#if IOS || MACCATALYST
+﻿using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
```
Comment (MattByers) [inline Raygun4Maui/MauiRUM/EventTrackers/Apple/RaygunUiViewControllerObserver.cs:2]: Was it causing any issues before, with this inside the if block?

Comment (ProRedCat) [reply Raygun4Maui/MauiRUM/EventTrackers/Apple/RaygunUiViewControllerObserver.cs:2]: Good point, this isn't actually used so we can remove it now :)

**Pull request merged**